addons/finger/stringtable.xml

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Project name="ace_dfinger">
+  <Package name="strings">
+
+
+
+
+
+
+	<Key ID="STR_Addons_ACE__dfinger__indicator_for_self">

```

Comment:
Key names should use macros to maintaing the same format as on the rest of the modules